web/app/scripts/views/record/add-edit-controller.js

```diff
@@ -123,13 +200,12 @@
                     data: editorData,
                     schema: ctl.recordSchema.uuid,
 
-                    // TODO: the following fields are external to the schema and need implementation
-                    // Generating bogus values for now -- to be revisited in a future task.
-                    slug: 'testslug', // Note: don't think we need a slug for a record, uuid is fine
-                    label: 'testlabel', // Note: label also seems unnecessarry for a record
-                    geom: 'POINT (0 0)', // TODO: we'll need a map with ability to search/drop point
-                    occurred_from: new Date(), // TODO: we'll need date pickers for occured from/to
-                    occurred_to: new Date()
+                    // constant fields
+                    slug: ctl.slug,
```

Comment:
I vote for removing the slug and the label from both this form and the backend (ashlar). If we do want to use a slug, it should be generated on the backend and would need to be unique: but I think the uuid works fine for this. I don't think the label will be useful in practice, since all of the information you'd want to put in a label will already be available in all of the other fields: it'll just make adding a new record more cumbersome.